pom.xml

```diff
@@ -54,19 +54,21 @@
         <version.deltaspike>1.9.0</version.deltaspike>
         <version.easymock>4.0.2</version.easymock>
         <version.eclipse.emf>2.15.0</version.eclipse.emf>
-        <version.geoserver>2.22.1</version.geoserver>
+        <version.geoserver>2.22.5</version.geoserver>
         <version.geotools>28.1</version.geotools>
         <version.geowave>1.2.0</version.geowave>
         <version.google-guava>31.1-jre</version.google-guava>
         <version.googlecode-findbugs>2.0.3</version.googlecode-findbugs>
         <version.googlecode-json-simple>1.1.1</version.googlecode-json-simple>
+        <version.guava>32.0.0-android</version.guava>
```

Comment:
Where is this being used?

Reply:
nowhere!  pulled it, and everything builds and tests run.